Populating data locally fails

The two main options to [populate data locally](https://github.com/GoogleChrome/webstatus.dev/blob/main/DEVELOPMENT.md#populate-data-locally) fail for me:
- `make dev_workflow` (or `make web_feature_local_workflow`) fails with:
```
go run ./util/cmd/local_web_feature_workflow/main.go -web_consumer_host=http://localhost:8092
util/cmd/local_web_feature_workflow/main.go:25:2: no required module provides package github.com/GoogleChrome/webstatus.dev/lib/gen/openapi/workflows/steps/web_feature_consumer: go.mod file not found in current directory or any parent directory; see 'go help modules'
make: *** [Makefile:357: web_feature_local_workflow] Error 1
```

Similarly `make dev_fake_data` fails with:
```
SPANNER_EMULATOR_HOST=localhost:9010 DATASTORE_EMULATOR_HOST=localhost:8085 go run ./util/cmd/load_fake_data/main.go -spanner_project=local -spanner_instance=local -spanner_database=local -datastore_project=local
util/cmd/load_fake_data/main.go:27:2: no required module provides package cloud.google.com/go/spanner: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:28:2: no required module provides package github.com/GoogleChrome/webstatus.dev/lib/gcpspanner: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:29:2: no required module provides package github.com/GoogleChrome/webstatus.dev/lib/gds: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:30:2: no required module provides package github.com/GoogleChrome/webstatus.dev/lib/gen/openapi/backend: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:31:2: no required module provides package github.com/brianvoe/gofakeit/v7: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:32:2: no required module provides package github.com/web-platform-tests/wpt.fyi/shared: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:33:2: no required module provides package golang.org/x/text/cases: go.mod file not found in current directory or any parent directory; see 'go help modules'
util/cmd/load_fake_data/main.go:34:2: no required module provides package golang.org/x/text/language: go.mod file not found in current directory or any parent directory; see 'go help modules'
make: *** [Makefile:369: dev_fake_data] Error 1
```

I managed to populate my data store running:
```
cd util/cmd/local_web_feature_workflow/
go run main.go -web_consumer_host=http://localhost:8092
```

--- a/Makefile
+++ b/Makefile
@@ -353,7 +353,7 @@ clean-node:
 ################################
 dev_workflows: bcd_workflow web_feature_local_workflow wpt_workflow
 web_feature_local_workflow: FLAGS := -web_consumer_host=http://localhost:8092
-web_feature_local_workflow: port-forward-manual
+web_feature_local_workflow: build port-forward-manual
 	go run ./util/cmd/local_web_feature_workflow/main.go $(FLAGS)
 wpt_workflow:
 	./util/run_job.sh wpt-consumer images/go_service.Dockerfile workflows/steps/services/wpt_consumer \
@@ -361,7 +361,7 @@ wpt_workflow:
 bcd_workflow:
 	./util/run_job.sh bcd-consumer images/go_service.Dockerfile workflows/steps/services/bcd_consumer \
 		workflows/steps/services/bcd_consumer/manifests/job.yaml bcd-consumer
-dev_fake_data: is_local_migration_ready
+dev_fake_data: build is_local_migration_ready
 	fuser -k 9010/tcp || true
 	kubectl port-forward --address 127.0.0.1 pod/spanner 9010:9010 2>&1 >/dev/null &
 	fuser -k 8085/tcp || true

